src/twitter.ts

```diff
@@ -62,6 +58,7 @@ const fetchUserId = async (
     },
   }).then((res) => res.json()) as any;
 
+  // Error handling
```

Comment:
Can you add a separate comment for each of these if statements? The first one handles unexpected errors, and the second handles there case where the user does not exist.